src/site/stages/build/process-cms-exports/schemas/transformed/node-vamc_operating_status_and_alerts.js

```diff
@@ -0,0 +1,67 @@
+const { usePartialSchema } = require('../../transformers/helpers');
+const localFacilitySchema = require('./node-health_care_local_facility');
+
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-vamc_operating_status_and_alerts'] },
+    entityType: { enum: ['node'] },
+    entityBundle: { enum: ['vamc_operating_status_and_alerts'] },
+    title: { type: 'string' },
+    entityPublished: { type: 'boolean' },
+    entityMetatags: { $ref: 'MetaTags' },
+    entityUrl: { $ref: 'EntityUrl' },
+    fieldBannerAlert: {
+      type: 'array',
+      items: { $ref: 'transformed/node-full_width_banner_alert' },
+      // maxItems: 1 // Presumably? The template expects an array, though...
```

Comment:
So should this be uncommented?

Reply:
:man_shrugging: Depends on how it's used in the template. I didn't know, so I left it open.